src/commands/fun/DivorceCommand.js

```diff
@@ -24,11 +24,11 @@ module.exports = class DivorceCommand extends Command {
     const user2Mention = await this.client.users.fetch(ctx.data.user.casado);
 
     ctx.send(`${ctx.locale('commands:divorce.confirmation')} ${user2Mention}`).then((msg) => {
-      msg.react('✅');
-      msg.react('❌');
+      msg.react(this.client.constants.emojis.yes);
+      msg.react(this.client.constants.emojis.no);
 
-      const filterYes = (reaction, usuario) => reaction.emoji.name === '✅' && usuario.id === ctx.message.author.id;
-      const filterNo = (reação, u) => reação.emoji.name === '❌' && u.id === ctx.message.author.id;
+      const filterYes = (reaction, usuario) => reaction.emoji.name === this.client.constants.emojis.yes && usuario.id === ctx.message.author.id;
+      const filterNo = (reação, u) => reação.emoji.name === this.client.constants.emojis.no && u.id === ctx.message.author.id;
```

Comment:
```suggestion
      const filter = (reaction, user) => reaction.me && user.id === ctx.message.author.id;
```
https://discord.js.org/#/docs/main/stable/class/MessageReaction?scrollTo=me
`.me` retorna `true` se o emoji que a pessoa reagiu, já foi reagida pelo bot

Reply:
O problema disso é que não vou poder diferenciar se foi escolhido o emoji de negação ou o de aprovação

Reply:
vc vai essa verificação dentro do collector, lembrando que o collector é um evento, esses dois eventos poderiam ser um só
Exemplo de uso: https://github.com/Simplicity/Simplicity/blob/57fd71c6ef66ab38a8d1b706f0c0dcc9f3b19bad/src/commands/guild/serverinfo.js#L105-L119